lib/logstash/environment.rb

```diff
@@ -114,6 +95,42 @@ def bundler_setup!(options = {})
       ::Bundler.setup
     end
 
+    def runtime_jars_root(dir_name, package)
```

Comment:
I agree with you in part, however I did it this way in order to provide more flexibility, keep in mind now this file has a ```ROOT_DIR``` just one level below the file that load the jars, but it can happen to be in another location. Or that for some jars, we aim to store them in another place. 

In my opinion, even if It might make sense, I find it too restrictive.